core/lib/mcxtx.h

```diff
@@ -256,5 +256,11 @@
      * bytes.
      */
 #    endif
-    dr_zmm_t simd[MCXT_NUM_SIMD_SLOTS];
+    union {
+        dr_zmm_t simd[MCXT_NUM_SIMD_SLOTS];
```

Comment:
These need their own doxygen comments as the existing one is now attached solely to the union.  You can see what the existing code above is doing: please follow that lead.  Feel free to update the existing comment lines 222-226 while doing that -- is that stale now?

Reply:
Done.
222-226 is not stale.